beacon-chain/p2p/encoder/ssz_test.go

```diff
@@ -8,13 +8,14 @@ import (
 	"math"
 	"testing"
 
-	"github.com/gogo/protobuf/proto"
+	gogo "github.com/gogo/protobuf/proto"
```

Comment:
why use gogo here ? aren't we trying to remove it

Reply:
This uses a function `gogo.EncodeVarint`, which I could not find in the standard proto library. Even if this uses gogo, its not as crucial as anything related to the APIs. Can look into finding a more native alternative for this in a future PR.

Reply:
It's here: https://github.com/golang/protobuf/blob/ae97035608a719c7a1c1c41bed0ae0744bdb0c6f/proto/buffer.go#L26

Let's remove gogo as much as possible, it is not maintained.

Reply:
Thats protobuf v1, but maybe its good since its a switch away from gogo, thanks

Reply:
seems like you are still using gogo ?

Reply:
I tried the change but it led to test failures here:
https://buildkite.com/prysmatic-labs/prysm/builds/50217

Not sure why....

Reply:
Spoke to preston, we can address this in a future PR. Would rather this PR just focus on API related stuff.